Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1255,6 +1255,9 @@
 /* Label text in device manager for a device with no name */
 "UNNAMED_DEVICE" = "Unnamed Device";
 
+/* Named used for groups without a name. */
+"UNNAMED_GROUP_NAME" = "Unnamed Group";
```

Comment:
Should we use this instead of "New Group" everywhere we display a group name, outside of the "group creation" view? Otherwise it might be confusing to see "Unnamed Group" in the notification which corresponds to some "New Group" in the inbox.